UnityProject/Assets/Scripts/Systems/StationObjectives/StationObjectiveManager.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Text;
+using System.Linq;
+using DiscordWebhook;
+using DatabaseAPI;
+using Messages.Server.LocalGuiMessages;
+using Strings;
+using Managers;
+using Random = UnityEngine.Random;
+
+namespace StationObjectives
+{
+	public class StationObjectiveManager : MonoBehaviour
+	{
+
+		[SerializeField]
+		[Tooltip("Stores all station objective data.")]
+		private StationObjectiveData stationObjectiveData = null;
+
+		public static StationObjectiveManager Instance;
+
+		[NonSerialized] public StationObjective activeObjective;
+
+		private void Awake()
+		{
+			if (Instance == null)
+			{
+				Instance = this;
+			}
+			else
+			{
+				Destroy(gameObject);
+			}
+		}
+		void OnEnable()
+		{
+			EventManager.AddHandler(EVENT.RoundEnded, ResetObjectives);
+		}
+		void OnDisable()
+		{
+			EventManager.RemoveHandler(EVENT.RoundEnded, ResetObjectives);
+		}
+
+		//public ;
+
+		public void RemindStation()
```

Comment:
this method is never called

Reply:
neither is the remind antagonist method but it's included there anyway just in case >:F